README.md

```diff
@@ -64,7 +66,8 @@ advanced one like this:
 	"compose-environment": {
 		"COMPOSE_PROJECT_NAME": "blog",
 		"COMPOSE_HTTP_TIMEOUT": "600"
-	}
+	},
+	"azure-environment": "AzureChinaCloud"
```

Comment:
@lizzha don't change this please. this is some example people will copy/paste. the docs you added above are sufficient.